app/components/organization/Pipeline/bar.js

```diff
@@ -105,3 +112,13 @@ export default class Bar extends React.Component {
     }
   }
 }
+
+export default Relay.createContainer(Bar, {
```

Comment:
I wonder if this is any problem for performance? I know @keithpitt was scared of so many Relay containers being created, and all the `getFragment` calls? Is there such a thing as "too many Relay containers"? It'd be good to find out…